recipes/restbed/all/conanfile.py

```diff
@@ -0,0 +1,128 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, load, rm, save
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+import os
+import re
+
+required_conan_version = ">=1.52.0"
+
+
+class RestbedConan(ConanFile):
+    name = "restbed"
+    homepage = "https://github.com/Corvusoft/restbed"
+    description = "Corvusoft's Restbed framework brings asynchronous RESTful functionality to C++14 applications."
+    topics = ("restful", "server", "client", "json", "http", "ssl", "tls")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "AGPL-3.0-or-later", "CPL"  # Corvusoft Permissive License (CPL)
```

Comment:
https://spdx.org/licenses/CPL.html says 404 🤔 

we are doing https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-license-should-i-use-for-a-custom-project-specific-license now 🙏

Reply:
```suggestion
    license = "AGPL-3.0-or-later", "LicenseRef-CPL"  # Corvusoft Permissive License (CPL)
```